README.rst

```diff
@@ -76,23 +82,50 @@ trailing ``:2.7`` makes sure we are using Python 2.7.
 4. Analysis workflow
 ====================
 
-The hello world example is simple enough in that it does not require any complex
-workflow steps to run. It consists of running a single command only.
-Nevertheless, it is represented in a structured YAML manner with `Yadage
-<https://github.com/diana-hep/yadage>`_ and `Common Workflow Language
-<http://www.commonwl.org/v1.0/>`_ specifications. The corresponding workflow
-descriptions can be found here:
+This analysis is very simple because it consists basically of running only the
+Python executable which will produce a text file.
+
+The workflow can be represented as follows:
+
+.. code-block:: console
+
+                START
+                |
+                |
+                V
+    +-------------------------------+
+    | run Python program            | <-- inputfile
```

Comment:
Maybe kill "run Python program" and call it simply `python helloworld.py`. It is shorter and self-explanatory.

Also  maybe let's expand input output files to use `names.txt` and `greetings.txt` since it will be more consistent, so we could have only `names.txt` and `sleeptime` coming from the right, and `greetings.txt` coming out.